test/e2e/framework/timeouts.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import "time"
+
+const (
+	// Default timeouts to be used in TimeoutContext
+	podStartTimeout            = 5 * time.Minute
+	podStartShortTimeout       = 2 * time.Minute
+	podDeleteTimeout           = 5 * time.Minute
+	claimProvisionTimeout      = 5 * time.Minute
```

Comment:
minor nit: its more readable to keep the timeouts for a given resource together. e.g claimProvisionTimeout, claimProvisionShortTimeout, claimBoundTimeout